src/Config.js

```diff
@@ -0,0 +1,6 @@
+// 여기가 찐
+const Config = {
+    API : "http://10.58.1.217:8000/product/sandwich",
+}
+
+export default Config
```

Comment:
이 방법도 괜찮지만 엔드포인트를 포함한 여러개의 변수를 항상 가지고 있어야한다는 단점이 있습니다.
```suggestion
export const URL = "http://10.58.1.217:8000";
```
위 처럼 선언해주시고 import 할 때 `import { URL } from 경로` 라고 해주시면 항상 정해진 변수뒤에 template literal 형태로 endpoint만 적어주시면 됩니다.
``` js
fetch(`${URL}/endpoint명`);
```